HandlePaymentUpdateFailures/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "type": "queueTrigger",
+      "direction": "in",
+      "name": "messageviewfailure",
+      "queueName": "%MESSAGE_VIEW_PAYMENT_UPDATE_FAILURE_QUEUE_NAME%",
+      "connection":"INTERNAL_STORAGE_CONNECTION_STRING"
+    }
+  ],
+  "retry": {
+    "strategy": "exponentialBackoff",
+    "maxRetryCount": 5,
```

Comment:
NB:
Queue triggers already have a default hidden retry set to 5.
With the function retry se to 5 too, this function will be retried for a max of 25 times.

Reply:
Ok so we can remove this config ?

Reply:
if 5 retry are enough, we can remote it.